homeassistant/components/template/__init__.py

```diff
@@ -109,7 +109,8 @@ async def init_coordinator(hass, conf_section):
                             "entities": conf_section[platform_domain],
                         },
                         hass_config,
-                    )
+                    ),
+                    eager_start=True,
```

Comment:
Same question as for mobile app PR.

Reply:
https://github.com/home-assistant/core/pull/114700#discussion_r1550175494